packages/fast-tooling-react/src/form/controls/control.children.tsx

```diff
@@ -39,6 +39,19 @@ class ChildrenControl extends React.Component<
         managedClasses: {},
     };
 
+    public static getDerivedStateFromProps(
+        props: ChildrenControlProps,
+        state: ChildrenControlState
+    ): Partial<ChildrenControlState> {
+        if (props.value !== state.data) {
+            return {
+                data: [].concat(props.value || []),
```

Comment:
if `this.state.data` is really just a shallow copy of `this.props.value`, do you really need to do this at all? props and state should never be mutated, so it seems like you could simply substitute `this.props.value` where you're using `this.state.data`

Reply:
We need it for dragging changes, since when you drag you need to show the updated state but the props arent updated until the drop is finished.